README.md

```diff
@@ -1 +1,86 @@
-# playwright
+# Redmine Testing Suite
+
+## Overview
+
+This repository contains a testing suite for the Redmine application, built using Playwright for end-to-end testing. The suite includes tests for user registration and login functionalities. The results of these tests are reported using Allure Report, which is displayed via GitHub Pages (gh-pages).
+
+## Getting Started
+
+### Prerequisites
+
+Ensure you have Node.js installed on your machine. You can download it from [nodejs.org](https://nodejs.org/).
+
+### Installation
+
+1. **Clone the repository:**
+
+   ```bash
+   git clone https://github.com/karenn32/playwright.git
+   cd playwright
+   ```
+
+2. **Install the dependencies:**
+
+   ```bash
+   npm install
+   ```
+
+3. **Install Playwright browsers:**
+
+   ```bash
+   npx playwright install
+   ```
+
+### Configuration
+
+Configure Playwright and Allure Report as needed in `playwright.config.js`.
```

Comment:
@karenn32 This one not really needed here, only short instruction - "How to set up" is required.